modelservice/src/test/java/com/expedia/adaptivealerting/modelservice/test/ObjectMother.java

```diff
@@ -0,0 +1,64 @@
+package com.expedia.adaptivealerting.modelservice.test;
+
+import com.expedia.adaptivealerting.modelservice.entity.Metric;
+import com.expedia.adaptivealerting.modelservice.providers.graphite.GraphiteResult;
+import com.expedia.adaptivealerting.modelservice.providers.graphite.Tags;
+import com.expedia.adaptivealerting.modelservice.service.AnomalyRequest;
+import com.expedia.adaptivealerting.modelservice.spi.MetricSource;
+import com.expedia.adaptivealerting.modelservice.spi.MetricSourceResult;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ObjectMother {
+    private static final ObjectMother MOM = new ObjectMother();
+
+    public static ObjectMother instance() {
+        return MOM;
+    }
+
+    private ObjectMother() {
+    }
+
+    public GraphiteResult[] getGraphiteData() {
+        GraphiteResult[] results = new GraphiteResult[1];
+        GraphiteResult result = new GraphiteResult();
```

Comment:
Stylistically, try to keep a declaration close to the code that actually uses it. For example here is would be clearer to build the datapoints first, then declare and build the result, and finally declare and build the results.

Like I say this is stylistic guidance, so it's your call.